docs/man/monerod.1

```diff
@@ -0,0 +1,5343 @@
+Allow any peer certificate
+.TP
+\fB\--disable-rpc-ban\fR
+Do not ban hosts on RPC errors
+.TP
+\fB\--rpc-payment-address arg\fR
+Restrict RPC to clients sending micropayment to this address
+.TP
+\fB\--rpc-payment-difficulty\fR arg (=1000)
+Restrict RPC to clients sending micropayment at this difficulty
+.TP
+\fB\--rpc-payment-credits\fR arg (=100)
+Restrict RPC to clients sending micropayment, yields that many credits per payment
+.TP
+\fB\--rpc-payment-allow-free-loopback\fR
+Allow free access from the loopback address (ie, the local host)
+
+
+.SS Monero notifications:
+.TP
+\fB\--block-notify\fR arg
+Run a program for each new block, '%s' will be replaced by the block hash
+.TP
+\fB\--block-rate-notify\fR arg
+Run a program when the block rate undergoes large fluctuations. This might be a sign of large amounts of
+hash rate going on and off the Monero network, and thus be of potential interest in predicting attacks. %t will be replaced by the number of minutes
+for the observation window, %b by the number of blocks observed within that window, and %e by the number of blocks that was expected in that window. It is
+suggested that this notification is used to automatically increase the number of confirmations required before a payment is acted upon.
+.TP
+\fB\--reorg-notify\fR arg
+Run a program for each reorg, '%s' will be replaced by the split height, '%h' will be replaced by the new blockchain height, '%n' will be replaced by the number of new blocks in the new chain,
+and '%d' will be replaced by the number of blocks discarded from the old chain
+
+
+
+
+.SS Mining
+.TP
+\fB\--start-mining\fR arg
+Specify wallet address to mining for
+.TP
+\fB\--mining-threads\fR arg
+Specify mining threads count
+.TP
+\fB\--bg-mining-enable\fR
+enable background mining
+.TP
+\fB\--bg-mining-ignore-battery\fR
+if true, assumes plugged in when unable to query system power status
+.TP
+\fB\--bg-mining-min-idle-interval\fR arg
+Specify min lookback interval in seconds for determining idle state
+.TP
+\fB\--bg-mining-idle-threshold\fR arg
+Specify minimum avg idle percentage over lookback interval
+.TP
+\fB\--bg-mining-miner-target\fR arg
+Specify maximum percentage cpu use by miner(s)
+
+
+.SS Testing
+.TP
+\fB\--keep-alt-blocks\fR
+Keep alternative blocks on restart
+.TP
+\fB\--test-dbg-lock-sleep\fR arg (=0)
+Sleep time in ms, defaults to 0 (off), used to debug before/after locking mutex. Values 100 to 1000 are good for tests.
+.TP
+\fB\--test-drop-download\fR
+For net tests: in download, discard ALL blocks instead checking/saving them (very fast)
+.TP
+\fB\--test-drop-download-height\fR arg (=0)
+Like test-drop-download but discards only after around certain height
+.TP
+\fB\--keep-fakechain\fR
+Don't delete any existing database when in fakechain mode.
+.TP
+\fB\--regtest\fR
+Run in a regression testing mode.
+.TP
+\fB\--fixed-difficulty\fR arg (=0)
+Fixed difficulty used for testing.
+
+
+.SS Legacy Options
+.TP
+\fB\--ban-list\fR arg
```

Comment:
With the exception of fluffy-blocks, these are not "legacy options", and some, such as ban-list, are currently suggested best practice to defend against network snooping by fleets node proxies. I would suggest just folding all of these into the server config section, though if you want to keep [no-]fluffy-blocks here, that's fine.

Reply:
Probably pulled / transposed some info from old monerodocs.org 

docs.getmonero.org updated this info to properly categorize some of the so-called "legacy" options